saleor/checkout/core.py

```diff
@@ -192,23 +192,29 @@ def is_shipping_same_as_billing(self):
         return Address.objects.are_identical(
             self.shipping_address, self.billing_address)
 
-    def _save_address(self, address, is_billing=False, is_shipping=False):
+    def _save_address(self, address, is_billing=False, is_shipping=False,
+                      create_copy=False):
```

Comment:
Please split this by domain of concern (saving to user's address book versus saving to order).